concepts/05 UI Components/Form/01 Getting Started with Form/60 Modify the Form at Runtime.md

```diff
@@ -0,0 +1,186 @@
+You can change any form, editor, or item properties at runtime. In the code below, we modify the [readOnly](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#readOnly) property's value:
```

Comment:
I meant that the entire description will go under the accordion tabs, because otherwise the logical flow of information is disrupted.